.github/workflows/dart_frog_auth.yaml

```
@@ -18,11 +18,13 @@ on:
 
 jobs:
   build:
-    uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/dart_package.yml@v1
+    uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/dart_package.yml@v1.13.0
     with:
       working_directory: packages/dart_frog_auth
 
-  pana:
-    uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/pana.yml@v1
-    with:
-      working_directory: packages/dart_frog_auth
+  # TODO(erickzanardo): temporarirly disabled while we investigate what is wrong
+  # with the pana action.
+  # pana:
+  #   uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/pana.yml@v1.13.0
+  #   with:
+  #     working_directory: packages/dart_frog_auth
```

I didn't know there was something wrong with the action. Should this change be in a different PR? Either way, we don't need to comment it out we can simply add a condition that is always false to skip it, [here](https://github.com/VeryGoodOpenSource/very_good_flutter_plugin/blob/20e47783fd7d94259179c17fa56f869c1d9e696c/.github/workflows/my_plugin.yaml#L38) is an example.

```suggestion
  pana:
    uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/pana.yml@v1
    with:
      if: false # TODO(erickzanardo): temporarirly disabled while we investigate what is wrong with the pana action.
      working_directory: packages/dart_frog_auth
```